README/ReleaseNotes/v630/index.md

```diff
@@ -49,7 +49,9 @@ The following people have contributed to this new version:
  Matevz Tadel, UCSD/CMS,\
  [QuillPusher](https://github.com/QuillPusher), [Compiler Research Group](https://compiler-research.org/team/),\
  Vassil Vassilev, Princeton/CMS,\
- Wouter Verkerke, NIKHEF/Atlas,
+ Wouter Verkerke, NIKHEF/Atlas,\
+ Dante Niewenhuis, UvA and CERN EP-SFT,\
+ Silia Taider, CPE Lyon and CERN EP-SFT
```

Comment:
Do we usually sort these entries? Looking at the rest, it looks mostly sorted (not fully though)...

Reply:
Yes they should be sorted, thanks!